res/database/Default/user/speedminister.xml

```diff
@@ -527,19 +530,19 @@
         </programme>
 
         <!-- ***** Serie: Expeditionen ins Tierreich, Dokumentation (Originaltitel der Episoden nicht mehr verfügbar) (real) ***** -->
-        <programme id="SpeedMinister-Serie-Expeditionen-01" product="4" licence_type="3" creator="8605" created_by="SpeedMinister">
+        <programme id="SpeedMinister-Serie-Expeditionen-01" product="4" licence_type="3" imdb_id="tt0305025" creator="8605" created_by="SpeedMinister" comment="The series started in 1965. As this programme was tagged with 1985, contributors are supposedly welcome to add more 'seasons'.">
             <title>
-                <de>Ausflüge ins Tierreich</de>
-                <en>Excursions into the animal kingdom</en>
+                <de>Ausflüge ins Tierreich (1985)</de>
```

Comment:
shorten to "'85"?

Reply:
In my upcoming data items I have a lot of series with seasons, so additions to the title like "(Season 1)" will become a common sight. (Although that could be abbreviated to "(S1)", "(S2)" too.)

Reply:
If Ronny does not intervene, 1985 insted of '85 is fine with me as well.

Reply:
'85 etc sounds good for concerts, or stuff like "WrestleParty '81"
Stuff with a kind of "interval" (each year, or decade or ... soccer, olympia).

If it references a "irregular" sequel, then whole year looks better.